src/biokbase/narrative/jobs/job.py

```diff
@@ -396,24 +399,17 @@ def output_state(self, state=None, no_refresh=False) -> dict:
         }
         :rtype: dict
         """
-        if not state:
-            state = self.cached_state() if no_refresh else self.refresh_state()
-        else:
-            self.update_state(state)
-            state = self.cached_state()
-
-        if state is None:
-            return self._create_error_state(
-                "Unable to find current job state. Please try again later, or contact KBase.",
-                "Unable to return job state",
-                -1,
-            )
-
+        state = self.cached_state()
         self._trim_ee2_state(state, OUTPUT_STATE_EXCLUDED_JOB_STATE_FIELDS)
+
         if "job_output" not in state:
             state["job_output"] = {}
-        for arg in EXTRA_JOB_STATE_FIELDS:
-            state[arg] = getattr(self, arg)
+
+        if "batch_id" not in state:
+            state["batch_id"] = self.batch_id
+
+        if "child_jobs" not in state:
+            state["child_jobs"] = JOB_ATTR_DEFAULTS["child_jobs"]
```

Comment:
access via `JOB_ATTR_DEFAULTS` or otherwise there'll be an unnecessary call to ee2